Formula/mpv.rb

```diff
@@ -0,0 +1,57 @@
+class Mpv < Formula
+  desc "Media player based on MPlayer and mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/mpv-player/mpv/archive/v0.30.0.tar.gz"
+  sha256 "33a1bcb7e74ff17f070e754c15c52228cf44f2cefbfd8f34886ae81df214ca35"
+  head "https://github.com/mpv-player/mpv.git"
+
+  depends_on "docutils" => :build
+  depends_on "pkg-config" => :build
+  depends_on "python" => :build
+
+  depends_on "ffmpeg" => "4.0"
+  depends_on "jpeg"
+  # uses_from_macos "libarchive" does not work
+  depends_on "libarchive"
+  depends_on "libass"
+  depends_on "little-cms2"
+  depends_on "lua@5.1"
```

Comment:
It doesn't build with latest lua version (5.3)?

Reply:
unfortunately, mpv does not support lua 5.3 at this point https://github.com/mpv-player/mpv/issues/5205